src/.clang-tidy

```diff
@@ -0,0 +1,82 @@
+# This .clang-tidy file is used by CI to ensure that commits do not worsen the
+# codebase. If you want to improve the codebase this config is not for you.
```

Comment:
Isn't this config for me if I want to improve the codebase by enabling some checks or even lower or raise some limits?

Reply:
```suggestion
# codebase. The checks and values below are the minimum standard for new code.
(Without claiming that they are 100% correct. They can be modified on demand!)
If you want to improve the codebase try enabling some additional checks or
playing with the configuration values.
```